### ISSUE TITLE
A11y_VSCodespaceBenchmark_VS-DeleteCodespace_ScreenReader: Narrator is not announcing the status after deleting the codespace.

### ISSUE THREAD

**Issue by khansahab15** [microsoft/vscode#114608]
**GitHubTags:**#A11yWCAG2.1; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCodespaceBenchmark;#WCAG4.1.3; #A11ySev2;#Benchmark;#Visual Studio Codespace;

["Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
**Environment Details:​**
Application Name: VS Codespace
Operating System: Win10​
Version: 1.47.0-insider (user setup)
Date: 2020-07-08T10:52:52.439Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041
NVDA version: 2020.1

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Codespace"
3. Click on "Visual Studio codespace".
4.Navigate till "Remote Explorer" control.
5.Navigate till "+" button beside the create new codespace button and click on it.
6.Create the new codespace by following the steps.
7.Launch Narrator.
8.Navigate up to created codespace, right click on it and select delete the codespace option.
9.Select yes in delete codespace pop up and observed that narrator is announcing the status like "Codespace deleted" or not.

**Actual:**
Narrator is not announcing the status after deleting the codespace.

**Expected: ​**
After deleting the codespace, narrator should announce the status like codespace deleted.

**​User Impact: ​**
Screen reader only users will face difficulty to understand the status.

**Attachment:**

https://user-images.githubusercontent.com/25275622/105131475-02877180-5b0f-11eb-93e9-0a788d126f64.mp4

**Comment by isidorn** [comment]
@alexr00 we do not have that API, however I have already heard requests like that and due to that I just created this item for next milestone: https://github.com/microsoft/vscode/issues/114718

**Comment by alexr00** [comment]
Excellent, thanks @isidorn. Once we have API for this, it will be up to the extension to appropriately use it. Thus, moving this issue to the Codespaces extension.
